app/views/devise/registrations/start.html.slim

```diff
@@ -0,0 +1,26 @@
+- title t('upaya.titles.registrations.start')
+
+
+= render 'devise/shared/auth_nav'
+.clearfix.pt1
+  .col.col-2.center
+    = image_tag(asset_url('check-outline.svg'), class: 'mt1 sm-m0')
+  .col.col-10
+    p.pb2.sm-pb3.mb2.sm-mb3.border-bottom
+      'Please use this site to securely login to uscis.gov.
```

Comment:
yes, agency should be dynamic, but for now i can use more generic language; localizing now